pandas/core/indexes/base.py

```diff
@@ -681,7 +681,44 @@ def _values(self):
         return self.values
 
     def get_values(self):
-        """ return the underlying data as an ndarray """
+        """
+        Return `Index` data as an `ndarray`.
+
+        Because this is a wrapper function around Index.values, it is useful
+        when one explicitly wants to use a getter.
+
+        Returns
+        -------
+        numpy.ndarray
+            A one-dimensional numpy `array` of the indexes.
```

Comment:
It looks like MultiIndex.get_values inherits this docstring. Can you check the dimensionality of it? If it's a 2-D array I would note that.

Reply:
I just checked, it is 1-D.
```python
In [5]: midx = pd.MultiIndex.from_arrays([[1, 2, 3], ['a', 'b', 'c']], names =  ('number', 'letter'))
In [6]: midx.ndim
Out[6]: 1
```

Reply:
Huh, that is not what I would have expected :) Could you add that as an example?

Reply:
Although, I guess that makes sense since it's just `self.values`